controller/gameRoomController.js

```diff
@@ -15,4 +15,11 @@ async function createGameRoomFromGame(req, res) {
   res.status(201).json(gameRoom);
 }
 
-module.exports = { gameRoomList, generateGameRooms, createGameRoomFromGame };
+async function getGameById(req, res) {
+  const gameRoom = await GameRoomService.findById(req.params.roomId);
```

Comment:
Eg ser at du bruker PascalCase på servicen her, men camelCase på gameModeService lenger nede. Eg stemmer for å bruke sistnemnde, sidan vi "ish" bruker ein instans av ein service, sjølv om vi eigentleg berre importerer ein konstant